packages/api/.env.template

```diff
@@ -34,3 +34,16 @@ JET_LOGGER_MODE=CONSOLE
 JET_LOGGER_FILEPATH=jet-logger.log
 JET_LOGGER_TIMESTAMP=TRUE
 JET_LOGGER_FORMAT=LINE
+
+
+## Settings ##
+NAME=Praise

```

Comment:
We don't use env file to set these defaults anymore.

Reply:
@nebs-dev it looks like these unused ENV values are still in `.env.template` -- can you remove them?